products/accesscontextmanager/api.yaml

```diff
@@ -571,6 +571,174 @@ objects:
                   - !ruby/object:Api::Type::String
                     name: 'location'
                     description: String indicating the location of the expression for error reporting, e.g. a file name and a position in the file      
+  - !ruby/object:Api::Resource
+    name: 'AccessLevelCondition'
+    create_url: "{{access_level}}"
+    base_url: ""
```

Comment:
Does this need to be specified?

Reply:
I don't believe so - there's a comment on the AccessLevel resource that it is empty on purpose. I'll copy that comment here as well.